src/components/card/card.css

```diff
@@ -138,3 +155,110 @@
 	flex: 1;
 	flex-wrap: wrap;
 }
+
+/* Vertical cards */
+
+.card--vertical {
+	max-width: 33rem;
+}
+
+.card--vertical .card__content {
+	flex-flow: column wrap;
+	text-align: left;
+}
+
+/* ---ribbon ---- */
+
+.card__badge {
+	display: none;
+}
+
+.card--badge .card__badge {
+	display: block;
+}
+
+.ribbon {
+	position: absolute;
+	right: -5px;
+	top: -5px;
+	z-index: 1;
+	overflow: hidden;
+	width: 75px;
+	height: 75px;
+	text-align: right;
+}
+
+.ribbon span {
+	font-size: 10px;
+	font-weight: bold;
+	color: #fff;
+	text-transform: uppercase;
+	text-align: center;
+	line-height: 20px;
+	transform: rotate(45deg);
+	-webkit-transform: rotate(45deg);
+	width: 100px;
+	display: block;
+	background: #79a70a;
+	background: linear-gradient(#2989d8 0%, #1e5799 100%);
+	box-shadow: 0 3px 10px -5px rgba(0, 0, 0, 1);
+	position: absolute;
+	top: 19px;
+	right: -21px;
```

Comment:
same here rem conversion needed

Reply:
I have updated all the changes you suggested.